gitreceive/receiver/flynn-receive.go

```diff
@@ -176,11 +176,20 @@ Options:
 	if err := client.CreateRelease(release); err != nil {
 		log.Fatalln("Error creating release:", err)
 	}
-	if err := client.DeployAppRelease(app.Name, release.ID); err != nil {
-		log.Fatalln("Error deploying app release:", err)
+
+	logf := func(e *ct.DeploymentEvent) {
+		if e.Status == "running" {
```

Comment:
Reverse this check and return to reduce the indentation (i.e. `if e.Status != "running" { return }`)